tsl/test/expected/compression_conflicts.out

```diff
@@ -493,3 +493,22 @@ WHERE hypertable_name = 'compressed_ht' ORDER BY chunk_name;
             9 | _hyper_9_9_chunk
 (3 rows)
 
+-- test for disabling DML decompression
+SHOW timescaledb.enable_dml_decompression;
+ timescaledb.enable_dml_decompression 
+--------------------------------------
+ on
+(1 row)
+
+SET timescaledb.enable_dml_decompression = false;
+\set ON_ERROR_STOP 0
+-- Should error because we disabled the DML decompression
+INSERT INTO compressed_ht VALUES ('2022-01-24 01:10:28.192199+05:30', '6', 0.876, 4.123, 'new insert row')
+  ON conflict(sensor_id, time)
+DO UPDATE SET sensor_id = excluded.sensor_id , name = 'ON CONFLICT DO UPDATE' RETURNING *;
+ERROR:  INSERT with ON CONFLICT DO UPDATE is not enabled on compressed chunks
+INSERT INTO compressed_ht VALUES ('2022-01-24 01:10:28.192199+05:30', '6', 0.876, 4.123, 'new insert row')
+  ON conflict(sensor_id, time)
+DO NOTHING;
+ERROR:  INSERT with ON CONFLICT DO UPDATE is not enabled on compressed chunks
```

Comment:
I think you should have different message for each operation like:

"INSERT with ON CONFLICT DO UPDATE is not enabled on compressed chunks"  for ON CONFLICT DO UPDATE
"INSERT with ON CONFLICT DO NOTHING is not enabled on compressed chunks" for ON CONFLICT DO NOTHING

Reply:
Sure.